src/gui/application.cpp

```diff
@@ -539,6 +539,15 @@ void Application::setupConfigFile()
         return;
     }
 
+    const QStringList filesList =
+        !Utility::isWindows()
+        ? QDir(oldDir).entryList(QDir::Files).filter(QRegularExpression("^(?!.*\\.log).*$"))
+        : QDir(oldDir).entryList(QDir::Files);
```

Comment:
I am confused by this change
why do we need to special case windows OS ?

Reply:
Because of the way how the configPath is created in [src/libsync/configfile.cpp#L366](https://github.com/kaikli/nextcloud-desktop-client/blob/fix-7654/src/libsync/configfile.cpp#L366)

For Windows the whole config directory is moved to the data location. On all other platforms only the configuration is moved to the config location (`$XDG_CONFIG_DIR`) and the log files remain in the data location (`$XDG_DATA_DIR`).